src/pathwayvis/components/map/map.component.ts

```diff
@@ -413,16 +413,28 @@ class MapComponentCtrl {
     if (selected.method.id === 'fva' || selected.method.id === 'pfba-fva') {
 
       // const fvaData = reactionData;
-      const fvaData = _.pickBy(reactionData, (d) => Math.abs((d.upper_bound + d.lower_bound) / 2) > 1e-7);
+      const fvaData = reactionData;
+      for (const i in fvaData) {
```

Comment:
please try to come up with a functional approach that's also immutable. If you need some guidance, let me know.